File: .github/examples/showimage.cpp
#include "congl.h"
using namespace std;
using namespace ConGL;
using namespace eng2D;

int main() {
    Screen screen;
    screen.toggleAutosize(true);
    screen.setFont({5, 5});
    Layout layout(&screen);

    shapes::Sprite spr("/textures/examples/image.txr");
    layout.addFigure(&spr);

    screen.setSurfaceSize(spr.getSize());

    while (true) {
        layout.draw();
    }

    return 0;
}

File: PY_MODULE/examples/showimage.py
from ConGL import *

screen = Screen()  
screen.toggleAutosize(True)
screen.setFont((5, 5))

main = eng2D.Layout(screen)
img = eng2D.shapes.Sprite('image.txr')
main.addFigure(img)

screen.setSurfaceSize(img.getSize())

while True:
    # terminate when escape is pressed
    if keys.released(chr(0x1b)): 
        break
        
    main.draw(True)